assets/webauthn.js

```diff
@@ -5,7 +5,9 @@ document.addEventListener('DOMContentLoaded', function(){
 
   const { startAttestation, startAssertion } = SimpleWebAuthnBrowser;
 
-  elemBeginRegister && elemBeginRegister.addEventListener('click', async () => {
+  elemBeginRegister && elemBeginRegister.addEventListener('click', async (ev) => {
+
+    ev.preventDefault();
```

Comment:
Not having this sent me on a wild goose chase. For some reason firefox 'submits' the form if an error happened.